spec/lib/msf/ui/console/command_dispatcher/creds_spec.rb

```diff
@@ -71,9 +71,9 @@
               'Credentials',
               '===========',
               '',
-              'host  origin  service  public    private   realm  private_type  JtR Format',
-              '----  ------  -------  ------    -------   -----  ------------  ----------',
-              '                       thisuser  thispass         Password      '
+              'host  origin  service  public    private   realm  private_type  JtR Format  cracked_password',
```

Comment:
All of these changes are good but none of this seems to be testing the new functionality that you added in to make sure that it works in a variety of scenarios. I'd probably want to see some tests added to check this functionality given this code is going to be heavily utilized by anyone running the `creds` command.

Reply:
Oh, Sorry I thought I should add specs for this new functionality in the next separate PR. Ok So I will add them.

Reply:
I added the specs which tests the new functionality.